tests/benchmark_otanalytics.py

```diff
@@ -390,34 +390,18 @@ def use_case_provider_empty_filtered(
 
 class TestBenchmarkTrackParser:
     ROUNDS = 1
-    ITERATIONS = 1
+    ITERATIONS = 5
     WARMUP_ROUNDS = 0
 
     def test_load_15min(
         self,
         benchmark: BenchmarkFixture,
-        use_case_provider_empty: UseCaseProvider,
+        pandas_track_parser: OttrkParser,
         track_file_15min: Path,
     ) -> None:
-        use_case = use_case_provider_empty.provide_track_repository
         benchmark.pedantic(
-            use_case,
-            args=([track_file_15min], CURRENT_DATASET_TYPE),
-            rounds=self.ROUNDS,
-            iterations=self.ITERATIONS,
-            warmup_rounds=self.WARMUP_ROUNDS,
-        )
-
-    def test_load_15min_filtered(
```

Comment:
We don't need the filtered benchmark anymore?

Reply:
Yes that is correct. The `OttrkParser` does not wrap the `FilteredTrackDataset` decorator around the `TrackDataset` in `TrackParseResult`. It returns a regular `PandasTrackDataset`. It is only after adding the parsed tracks to the `TrackRepository` that the tracks are contained by a `FilteredTrackDataset`.